easybuild/tools/config.py

```diff
@@ -323,8 +325,15 @@ def install_path(typ=None):
         typ = 'modules'
 
     variables = ConfigurationVariables()
-    suffix = variables['subdir_%s' % typ]
-    return os.path.join(variables['installpath'], suffix)
+
+    res = None
+    if variables.get('installpath_%s' % typ, None) is None:
```

Comment:
no, that won't work, since the `installpath_software` key will always be there

I'll change it to not using `.get` and inverse the logic